From ee9744b2922fa5f4bb4f87ba2761adfc9a96d3a8 Mon Sep 17 00:00:00 2001
From: Nick Hollinghurst <nick.hollinghurst@raspberrypi.com>
Date: Fri, 28 Jul 2023 11:59:28 +0100
Subject: [PATCH 0572/1002] media: i2c: imx296: Add 2ms delay after releasing
 standby

The delay seems to be required to reliably read model ID.
(The same delay is already used when starting the camera.)

Signed-off-by: Nick Hollinghurst <nick.hollinghurst@raspberrypi.com>
---
 drivers/media/i2c/imx296.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/imx296.c b/drivers/media/i2c/imx296.c
index e16588f2791a..adce60a92002 100644
--- a/drivers/media/i2c/imx296.c
+++ b/drivers/media/i2c/imx296.c
@@ -949,6 +949,7 @@ static int imx296_identify_model(struct imx296 *sensor)
 			"failed to get sensor out of standby (%d)\n", ret);
 		return ret;
 	}
+	usleep_range(2000, 5000);
 
 	usleep_range(2000, 5000);
 
-- 
2.44.0

